pkg/util/controllerfinder/pods_finder.go

```diff
@@ -91,6 +91,7 @@ func (r *ControllerFinder) GetPodsForRef(apiVersion, kind, ns, name string, acti
 		labelSelector = obj.Selector
 		workloadUIDs = append(workloadUIDs, obj.UID)
 	}
+	klog.V(5).InfoS("find pods and replicas result", "workloadReplicas", workloadReplicas, "workloadUIDs", workloadUIDs, "labelSelector", labelSelector)
```

Comment:
The uid is not very suitable for viewing logs, please update it to be more log-friendly with some key.

Reply:
it is very useful to check whether the pods are listed properly while debugging